python/hsfs/core/external_feature_group_engine.py

```diff
@@ -31,6 +31,9 @@ def save(self, feature_group):
 
         # set primary and partition key columns
         # we should move this to the backend
+        util.verify_attribute_key_names(
+            "primary", feature_group.primary_key, feature_group.features
+        )
```

Comment:
Same comment as for the java part - external feature groups can also have event time columns